recipes/backward-cpp/all/conanfile.py

```diff
@@ -0,0 +1,152 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class BackwardCppConan(ConanFile):
+    name = "backward-cpp"
+    description = "A beautiful stack trace pretty printer for C++"
+    homepage = "https://github.com/bombela/backward-cpp"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("conan", "backward-cpp", "stack-trace")
+    license = "MIT"
+    exports_sources = [ "CMakeLists.txt", "patches/backward-cpp-*.patch" ]
+    generators = "cmake"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+       "stack_walking_unwind": [True, False],
+       "stack_walking_backtrace": [True, False],
+       "stack_details_auto_detect": [False], # dont let backtrace auto decide
+       "stack_details_backtrace_symbol": [True, False],
+       "stack_details_dw": [True, False],
+       "stack_details_bfd": [True, False],
+       "stack_details_dwarf": [True, False],
+       "shared": [True, False],
+       "fPIC": [True, False]
+    }
+    default_options = {
+       "stack_walking_unwind": True,
+       "stack_walking_backtrace": False,
+       "stack_details_auto_detect": False,
+       "stack_details_dw": False,
+       "stack_details_bfd": False,
+       "stack_details_dwarf": True,
+       "stack_details_backtrace_symbol": False,
+       "shared": True,
+       "fPIC": True
+    }
+
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+
+
+    def config_options(self):     
+          if self.settings.os == "Macos":
+            del self.options.stack_details_dw
+            del self.options.stack_details_bfd
+            del self.options.stack_details_dwarf
+    
+    def _check_options(self, options):
+        sum = 0
+        for name, value in self.options.values.as_list():
+            if value and name in options:
+                sum = sum + 1
+        if sum != 1:
+            return False
+        return True
+
+    def configure(self):
+        if self.settings.os not in ["Linux", "Macos", "Android"]:
+            raise ConanInvalidConfiguration("upstream backward-cpp v{0} is not \
+                supported in {1}.".format(self.version, self.settings.os))
+        # windows implementation only available in upstream master branch
+        
+        if not self._check_options(['stack_walking_unwind', 'stack_walking_backtrace']):
```

Comment:
I think it makes more sense to have an option called `"stack_walking": ["unwind", "backtrace"]`